README.md

```diff
@@ -197,7 +205,7 @@ connection.release(preparedStatement: preparedStatement) { result in
 ### Table creation
 Swift-Kuery enables you to create tables on the database server.
 
-Let's rewrite our `Grades` table by adding columns type and constraints: 
+Let's revisit the Grades table, which we used in our Example above, and add in two new columns containing the data type and constraints:
```

Comment:
Please can you check this section as now you've rewritten the example one of the "two new columns", the data type already exists now so this needs fixing up.